src/lib/db/PouchNoteDb.ts

```diff
@@ -178,10 +191,12 @@ export default class PouchNoteDb implements NoteDb {
           })
         )
       )
+      // todo: [komediruzecki-14. 07. 2021.] See if we need to handle POUCH DB noteId set and other things
```

Comment:
We should probably ignore pouch DB completely and force users to just tranfser storages prior to updating app to this release